docs/FAQ

```diff
@@ -557,8 +557,8 @@ FAQ
 
   3.9 How do I use curl in my favorite programming language?
 
-  There exist many language interfaces/bindings for curl that integrates it
-  better with various languages. If you are fluid in a script language, you
+  There exist many language interfaces/bindings for curl that integrate it
```

Comment:
This might read better phrased something lie:

Many language interfaces/bindings exist for curl that allow you to use it without having to use the command line tool. If you are fluent in such a script language you may prefer to use one of these interfaces instead.

Reply:
Thanks for looking everything over. I took a stab with this:

> Many programming languages have interfaces/bindings that allow you to use curl without having to use the command line tool. If you are fluent in such a language, you may prefer to use one of these interfaces instead.

Not sure if I'm supposed to click the `Resolve conversation` button or wait for you to approve and resolve?